app/src/main/java/com/hapramp/ui/activity/DetailedActivity.java

```diff
@@ -300,7 +300,18 @@ private void renderMarkdown(String body) {
     webView.getSettings().setPluginState(WebSettings.PluginState.ON);
     webView.setWebChromeClient(new WebChromeClient());
     webView.getSettings().setAllowFileAccess(true);
-    webView.setWebViewClient(new WebViewClient());
+    webView.setWebViewClient(new WebViewClient() {
+      @Override
+      public boolean shouldOverrideUrlLoading(WebView view, String url) {
+        if (url != null && (url.startsWith("http://") || url.startsWith("https://"))) {
```

Comment:
Use a regex validation for URLs here.
How about this one (cc @aviaryan)? -
```
(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?
```

Will this also override images (since they start with `https?://`)?

Reply:
Not sure `www.` goes there since we can have sub-domains too.
This might be a good solution. https://stackoverflow.com/a/14003815/2295672

Reply:
But that solution won't create a link if we have `www.google.com`.